cohorts/utils.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) 2016. Mount Sinai School of Medicine
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re 
+
+def _clean_column_name(x, keep_parens = True):
+    """
```

Comment:
Style: lines should be indented at the same level as the `"""`: https://www.python.org/dev/peps/pep-0008/#documentation-strings